ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -305,6 +305,38 @@ def test_source_ipahealthcheck_dogtag_ca_dogtagcertsconfigcheck(self):
         self.master.run_command(["mv", TOMCAT_CFG + ".old", TOMCAT_CFG])
         self.master.run_command(["ipactl", "restart"])
 
+    @pytest.fixture
+    def restart_tomcat(self):
+        """Fixture to Stop and then start tomcat instance during test"""
+        self.master.run_command(
+            ["systemctl", "stop", "pki-tomcatd@pki-tomcat"]
+        )
+        yield
+        self.master.run_command(
+            ["systemctl", "start", "pki-tomcatd@pki-tomcat"]
+        )
+
+    def test_ipahealthcheck_dogtag_ca_connectivity_check(self, restart_tomcat):
+        """
+        This testcase checks that when the pki-tomcat service is stopped,
+        DogtagCertsConnectivityCheck displays the result as ERROR
+        and check displays the result as SUCCESS when pki-tomcat
+        service is running on IPA Master.
```

Comment:
The comment is out-of-date (the test does not check SUCCESS).

Reply:
The doctsring has been updated.